requirements/pip.txt

```diff
@@ -69,47 +69,47 @@ selectolax==0.2.1
 orjson==2.0.7  # pyup: ignore
 
 # Utils
-django-gravatar2==1.4.2
+django-gravatar2==1.4.4
 pytz==2019.3
 Unipath==1.1
 django-kombu==0.9.4
-mock==3.0.5
-stripe==2.38.0
-regex==2019.11.1
+mock==4.0.1
+stripe==2.42.0
+regex==2020.1.8
 
 # unicode-slugify==0.1.5 is not released on PyPI yet
 git+https://github.com/mozilla/unicode-slugify@b696c37#egg=unicode-slugify==0.1.5
 
-django-formtools==2.1
-django-crispy-forms==1.8.0
+django-formtools==2.2
+django-crispy-forms==1.8.1
 
-docker==4.1.0
+docker==4.2.0
```

Comment:
This is scary -- I'd want to test it more.

Reply:
There is nothing breaking in the changelog https://docker-py.readthedocs.io/en/stable/change-log.html

Reply:
Changelogs can't possible know what might break with out particular setup. We've broken lots of things before because of upgrades.